src/components/TodoFooter/TodoFooter.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+import { Filter } from '../../types/Filter';
+
+type Props = {
+  todos: Todo[],
+  setFilterStatus: (filter: Filter) => void,
```

Comment:
if a component accepts a callback and calls it on some event, then prop is better to name it with a prefix `on`
```suggestion
  onChangeFilter: (filter: Filter) => void,
```